docs/usage/file-system.md

```diff
@@ -0,0 +1,48 @@
+(file-system)=
+
+# Dealing with Pyodide file system
+
+Pyodide supports file system through [Emscripten File System API](https://emscripten.org/docs/api_reference/Filesystem-API.html#filesystem-api).
+In JavaScript, Pyodide file system can be accessed through {any}`pyodide.FS`.
```

Comment:
```suggestion
Pyodide includes a file system provided by Emscripten.
In JavaScript, Pyodide file system can be accessed through {any}`pyodide.FS` which re-exports the  [Emscripten File System API](https://emscripten.org/docs/api_reference/Filesystem-API.html#filesystem-api)
```
maybe to give more credit to Emscripten

Reply:
Should be "*the* Pyodide file system can be accessed through ..."